classes/memos.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'pstore'
+
+class Memos
+  STORAGE_FILE = './tmp/storage'
+
+  def self.all
+    store = PStore.new(STORAGE_FILE)
+    store.transaction do
+      memos = store['memos'] || {}
+      memos.values
+    end
+  end
+
+  def self.find(id)
+    store = PStore.new(STORAGE_FILE)
+    store.transaction do
+      memos = store['memos'] || {}
+      memo = memos.select { |k| k == id }
+      memo.values.first
```

Comment:
ここは `Hash#select` を呼び出さなくても `Hash#[]` を呼び出せば、指定の id の memo が一発で取れるような気がします 🔥 

```suggestion
      memo = memos[id]
      memo
```

そうすると、そもそも変数に受ける必要がなくなるので、以下のように 1 行で書けるかもしれません。

```suggestion
      memos[id]
```

Reply:
確かにもっとシンプルにかけますね！